docs/release-notes/v1.50-v1.59.md

```diff
@@ -2,13 +2,310 @@
+
+* **Enhancements for VCF File Retrieval and Filtering. Search/Filter for Reference Genome by Variant Group**
+
+To enhance the usability of VCF-related endpoints, the 1.59 release allows users to search and filter variant groups by the associated reference genome. These improvements help ensure the appropriate reference genome is being used for gene variant analysis. The section **referenceGenome** is now displayed in the selected endpoints.
+
+* ??? danger "Affected endpoints (for both as-curator and as-users) that include referenceGenome"
+    * GET /api/v1/as-user/variants/group  
+    * GET /api/v1/as-curator/variants/group  
+    * GET /api/v1/as-user/variants/group/by/run/{id}  
+    * GET /api/v1/as-curator/variants/group/by/run/{id}  
+    * GET /api/v1/as-user/variants/group/{id}  
+    * GET /api/v1/as-curator/variants/group/{id}  
+    * GET /api/v1/as-user/omics/variant/group  
+    * GET /api/v1/as-curator/omics/variant/group
+    ![referenceGenome](1.59/referenceGenome.png)
+    <figcaption>Each response now includes a <strong>referenceGenome</strong> section that provides details about the associated reference genome</figcaption>
+
+* **Enhancements for VCF File Retrieval and Filtering. New Filtering Parameter: referenceGenomeId**
+
+A new query parameter, **referenceGenomeId** has been added to the omics endpoints to enable filtering by Reference Genome ID.
+
+* ??? danger "Affected endpoints"  
+    * GET /api/v1/as-user/omics/variant/group  
+    * GET /api/v1/as-curator/omics/variant/group
+    ![referenceGenomeId](1.59/referenceGenomeId.png)
+    <figcaption>The parameter <strong>referenceGenomeId</strong> has been added to selected endpoints, to allow retrieval of variant data filtering for specific Reference Genome ID</figcaption>
+
+#### [Curators]
+
+* **New endpoints for importing, and retrieving Reference Genomes**
+
+Users can now import different versions and builds of reference genomes. This feature allows users to significantly enhance flexibility, accuracy, and relevance in variant analysis.
+
+Uploading reference genomes was already supported in ODM but only via .py script. From this release (1.59), uploading of reference genomes is now available via a new API endpoint: <code>POST api/v1/reference-genomes</code>, part of the new definition **referenceGenome**.
+
+* ??? abstract "Important considerations of the Reference Genome endpoints"
+    * Only users that are part of the Curator Group can execute the endpoint <code>api/v1/reference-genomes</code> to upload and query reference genomes
+    * Each uploaded reference genome should be unique according to a combination of mandatory fields specified in a request body
+    * Reference genome requests should include the following parameters:
+    * | Field Name    | Required | Description |
```

Comment:
Could you please remove the asterisk sign? The item marker doesn't look good before the table.

Reply:
Disclaimer: I meant the asterisk